src/plugins/intel_gpu/src/graph/impls/onednn/utils.cpp

```diff
@@ -472,7 +385,7 @@ dnnl::algorithm convert_activation_func(cldnn::activation_func func) {
     switch (func) {
         case cldnn::activation_func::relu: return dnnl::algorithm::eltwise_relu;
         case cldnn::activation_func::relu_negative_slope: return dnnl::algorithm::eltwise_relu;
-        case cldnn::activation_func::gelu: return dnnl::algorithm::eltwise_gelu;
+        case cldnn::activation_func::gelu: return dnnl::algorithm::eltwise_gelu_tanh;
```

Comment:
we have two flavors of gelu from openvino spec. Why is it mapped to `gelu_tanh`?
https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/activation/GELU_7.md

Reply:
Update done.